src/vue/pages/Limits.vue

```diff
@@ -0,0 +1,270 @@
+<template>
+  <div class="limits" v-if="accountBalances.length">
+    <top-bar>
+      <template slot="main">
+        <router-link :to="vueRoutes.limits">
+          <span>{{ 'limits.explore-limits' | globalize }}</span>
+        </router-link>
+      </template>
+      <template
+        v-if="accountTypeI === ACCOUNT_TYPES.syndicate"
+        slot="extra"
+      >
+        <button
+          v-ripple
+          class="app__button-raised"
+          @click="isLimitsChangeDrawerShown = true"
+        >
+          {{ 'limits.change-limits-bth' | globalize }}
+        </button>
+      </template>
+    </top-bar>
+    <div class="limits__asset-selector">
+      <!--
+        :key is a hack to ensure that the component will be updated
+        after computed calculated
+      -->
+      <select-field
+        v-model="selectedAsset"
+        :values="accountBalancesAssetsCodes"
+        :form-free="true"
+        :key="`limits-asset-selector-${selectedAsset}`"
+        class="limits__assets-select"
+      />
+    </div>
+
+    <limits-table-renderer
+      :is-loading="isLimitsLoading"
+      :limits="selectedLimitsList"
+    />
+
+    <div class="limits__requests">
+      <h3 class="limits__requests-title">
+        {{ 'limits.requests-subheading' | globalize }}
+      </h3>
+
+      <limits-requests-list-renderer
+        :is-loading="isLimitsRequestsLoading"
+        :requests="limitsRequests"
+        @requests-reload-ask="reloadRequests"
+      />
+
+      <div class="limits__requests-collection-loader">
+        <!-- hack with key allows us to reload data -->
+        <collection-loader
+          :key="`request-reloader-${currentRequestsReloadingCount}`"
```

Comment:
we can avoid abusing hack with key here by reassigning first-page-loader, please see `/src/vue/pages/Operations.vue:76` for example.

Reply:
I think this (`/src/vue/pages/Operations.vue:76`) realization really strange. Right now I don't know how to make it better, but even `CollectionLoader.methods.reloadHandler()` looks better than the current version, is it?

**Fixed as in example**

Reply:
lgtm